src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/PandocCompiler.kt

```diff
@@ -0,0 +1,31 @@
+package com.bardsoftware.backend.fleet.rmarkdown
```

Comment:
Pls add copyright notice. There must be a setting for this (C)

Reply:
Fixed